src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -318,89 +326,70 @@ public List<JobModel> enumerateJobs(
         throw new IllegalArgumentException(String.format("Unrecognized direction %s", direction));
     }
 
-    // Filter on FQ class name if specified
-    if (!StringUtils.isEmpty(className)) {
-      filter.addFilterFlightClass(FlightFilterOp.EQUAL, className);
-    }
-
-    List<FlightState> flightStateList;
-    boolean canListAnyJob = checkUserCanListAnyJob(userReq);
+    List<FlightState> flightStateList = null;
     try {
-      if (canListAnyJob) {
-        flightStateList = stairway.getFlights(offset, limit, filter);
-      } else {
-        // Filter out sub-flights
-        filter.addFilterFlightClass(FlightFilterOp.NOT_EQUAL, FileIngestWorkerFlight.class);
-
-        flightStateList = getAccessibleFlights(offset, limit, filter, userReq);
-      }
-    } catch (InterruptedException ex) {
-      throw new JobServiceShutdownException("Job service interrupted", ex);
+      flightStateList = stairway.getFlights(offset, limit, filter);
```

Comment:
Since the logic is simpler now you can return directly here and don't need to declare `flightStateList`
```suggestion
      return stairway.getFlights(offset, limit, filter).stream().map(this::mapFlightStateToJobModel).toList();
```

Reply:
great call